src/core/manager.h

```diff
@@ -384,6 +384,10 @@ struct Manager {
         /* non-zero if we are reloading or reexecuting, */
         int n_reloading;
 
+        /* Reload timestamps */
+        usec_t reloading_start_monotonic_timestamp;
+        usec_t reloaded_realtime_timestamp;
```

Comment:
isn't MANAGER_TIMESTAMP_UNITS_LOAD_START what you actually want, and wat we already have?

Reply:
To me it looks like ``MANAGER TIMESTAMP UNITS_LOAD_START`` is updated only at boot, so I can't use it in this scenario.

Reply:
then add another timestamp to that array please